e2e/tests-dfx/network.bash

```diff
@@ -82,5 +83,6 @@ teardown() {
     setup_actuallylocal_shared_network
 
     assert_command_fail dfx build --network actuallylocal
-    assert_match "Cannot find canister id. Please issue 'dfx canister create e2e_project_frontend --network actuallylocal"
+    assert_match "Cannot find canister id."
+    assert_match "--network actuallylocal"
```

Comment:
The order of canister building is non-deterministic since it is sourced from a `HashSet`. This caused this test to fail on CI even though it passes locally.

The changes preserve what the test is supposed to test, but stops checking for the order in which canisters are checked